readthedocs/subscriptions/constants.py

```diff
@@ -20,6 +20,7 @@
 TYPE_AUDIT_PAGEVIEWS = "audit-pageviews"
 TYPE_REDIRECTS_LIMIT = "redirects-limit"
 TYPE_SSO_SAML = "sso-saml"
+TYPE_FASTER_BUILDS = "faster_builds"
```

Comment:
```suggestion
TYPE_FASTER_BUILDS = "faster-builds"
```

Looks like we are using hyphens for new variables